workshop.pl

```diff
@@ -1503,9 +1528,9 @@ sub arg_handler {
 logger('info', "succeeded\n", 1);
 command_logger('verbose', $command, $rc, $command_output);
 
-if (-e $container_mount_point . "/etc/resolv.conf." . $me) {
+if (-e $container_mount_point . "/etc/resolv.conf.workshop") {
     logger('info', "Restoring the backup of the temporary container's /etc/resolv.conf...\n");
-    ($command, $command_output, $rc) = run_command("mv --verbose " . $container_mount_point . "/etc/resolv.conf." . $me . " " . $container_mount_point . "/etc/resolv.conf");
+    ($command, $command_output, $rc) = run_command("/bin/cp --verbose --force " . $container_mount_point . "/etc/resolv.conf.workshop " . $container_mount_point . "/etc/resolv.conf");
```

Comment:
and this?

Reply:
I believe it is just the 'mv' that was problematic, and for some reason rm + cp instead seems to work more more reliably.  The full path on all of these is probably not necessary, just an extra precaution.  --force is also probably not necessary.

Reply:
Its not really important to this PR, but I sure wish I understand why you are getting better behavior from `rm+cp` compared to `mv`.